feg/protos/s6a_proxy.proto

```diff
@@ -94,6 +94,8 @@ message AuthenticationInformationRequest {
     uint32 num_requested_utran_geran_vectors = 6;
     // UTRAN/GERAN Resync Info (optional)
     bytes utran_geran_resync_info = 7;
+    // NR as secondary RAT bit in ULR message
+    bool supported_features_feature_id_list_2_nr_as_secondary_rat = 8;
```

Comment:
Per our discussion here https://github.com/magma/magma/issues/5672

shouldn't we use `Feature-List-ID-2 {NR as Secondary RAT}`

Reply:
Since potentially we can have more flags enabled from  `Feature-List-ID-2`  in the future, lets create a message for itself. Changing protos is very difficult once they are in place

```

message AuthenticationInformationRequest {
...
FeatureListId2 feature_list_id_2 = 8;
...
}

message FeatureListId2 {
	bool nr_as_secondary_rat = 1;
}

```

Reply:
per 
http://www.eventhelix.com/5G/non-standalone-access-en-dc/en-dc-secondary-node-addition-core-network-details.pdf
I see it uses `Update Location Request`, but I don't see `AuthenticationInformationRequest` Is it really necessary to include that field on this message? Maybe it is only necessary on UpdateLocationReques/Response?

Reply:
According to 3GPP 29.272 V-15.10.0, Supported features(as an Optional field) AVP should be added in Authentication Information Request. NR as secondary RAT is one of the bit field in Feature list ID2 which is a part of Supported features AVP.